rkt/api_service.go

```diff
@@ -51,7 +51,7 @@ var (
 
 func init() {
 	cmdRkt.AddCommand(cmdAPIService)
-	cmdAPIService.Flags().StringVar(&flagAPIServiceListenClientURL, "--listen-client-url", common.APIServiceListenClientURL, "address to listen on client API requests")
```

Comment:
hm, isn't this kind of a regression? not being able to configure the listen IP address is bad. similar discussion in https://github.com/coreos/rkt/issues/1267